sanic/exceptions.py

```diff
@@ -86,6 +87,7 @@ def __init__(
         self.status_code = status_code or self.status_code
         self.quiet = quiet
         self.headers = headers
+        self.message = message  # type: ignore
```

Comment:
There is a type error here — it looks like `message` the argument of `__init__` can take a `bytes` object, but the class attribute can only be a `str`.

Reply:
I suspect the option for it to be `bytes` might be because of `STATUS_CODES` messages being in bytes (assuming someone passed one of those to init), and this may be necessary to keep for compatibility.

Can you get away with typing without the ignore if you add some logic to decode the argument before storing to `self.message`, if `bytes` was received?

Reply:
Ah, that makes sense! I'll update my PR to do that :)